snippets/OtlpGrpcExporter_logs_kitchen_sink.yaml

```diff
@@ -0,0 +1,20 @@
+file_format: "1.0-rc.2"
+
+tracer_provider:
```

Comment:
the name of the file and the contents dont align here and in traces_kitchen_sink

Reply:
bah

Reply:
fixed